index.html

```diff
@@ -77,7 +77,7 @@
 			<div class="container">
 				<div class="row">
 					<div class="hidden-xs col-sm-2 col-md-2 col-lg-2">
-						<img src="images/Zoltu-Logo-Web.png" class="header-logo" />
+						<img src="images/bag-with-a-heart.png" class="header-logo" style="float:left;" /> <span style="font-size:20px; color:white;margin-top:18px;display:block">Bag<span style="color:#FFE117">Cupid</span></span>
```

Comment:
Did you intend to submit this line?  I would prefer to stick with the Zoltu logo until we settle on an official one.